src/shared/components/quoteBanner/quoteBanner.js

```diff
@@ -5,7 +5,7 @@ import styles from './quoteBanner.css';
 class QuoteBanner extends Component {
```

Comment:
While you're in here, can you take the opportunity to convert this to a functional component?

`const QuoteBanner = props => (JSX);`

Lastly, we lost the gray background that worked well on most of the pages, can you add an optional boolean prop for `isDark`. If `isDark`, the font should be white and the quoteBanner background should be the former steely gray. Once you've done that, you can throw `isDark: true,` onto line 89 of `src/scenes/home/team/team.js`.